src/AbortHandler.ts

```diff
@@ -28,60 +27,48 @@ export class AbortHandler implements LambdaInterface {
 		// clear PersistentLogAttributes set by any previous invocation, and add lambda context for this invocation
 		logger.setPersistentLogAttributes({});
 		logger.addContext(context);
-		switch (event.resource) {
-			case ResourcesEnum.ABORT:
```

Comment:
Can we remove ABORT from the enum now? Or are you planning to just delete the whole file eventually?

Reply:
Removed file

Reply:
Lovely